mlfinlab/ensemble/sb_bagging.py

```diff
@@ -475,43 +475,6 @@ def _validate_estimator(self):
         super(BaggingClassifier, self)._validate_estimator(
             default=DecisionTreeClassifier())
 
-    def _set_oob_score(self, X, y):
-        n_samples = y.shape[0]
-        n_classes_ = self.n_classes_
-
-        predictions = np.zeros((n_samples, n_classes_))
-
-        for estimator, samples, features in zip(self.estimators_,
-                                                self.get_estimators_samples_(),
```

Comment:
Are you sure you can delete this code entirely? I see that you are calling the custom method `self.get_estimators_samples_()` which is not present in the parent class.

Reply:
Any particular reason for leaving the underscore at the end of this method ? `get_estimators_samples_`